### PR TITLE
Phone dashboard redesign

### PR DESCRIPTION
**Pull request by vishk1**
Fixed Placement of Components:
- used flex grids to place components on the page
      - used large flex-row with 2 smaller flex-cols to split up the page
      - for left col, just added components
      - for right col: 
             - added components
             - split up into 2 flex-rows
             - top row: added 2 flex cols for newstrainingcard & leaderboard & sized them accordingly
             - bottom row: added and resized stories card

Misc:
- added temp newstrainingcard pic
- added temp stories card pic
- fixed path of temp logo in leadership card

### CODE REVIEW

**Comment by vishk1** [comment]
image

File: src/components/phone/CallerIDEditCard.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <base-button
+      variant="outline"
+      :action="
+        () => {
+          isShowingModal = true;
```
Comment (BradenM) [inline src/components/phone/CallerIDEditCard.vue]: :P

**Comment by vishk1** [comment]
i see now that the calls is slightly misplaced, i'll update that real quick

File: src/components/phone/CallerIDEditCard.vue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          Welcome back John!
+        </base-text>
+        <base-text
+          variant="h1"
+          weight="300"
+          class="text-crisiscleanup-dark-200"
+        >
+          Please confirm the number you would like to use today. <br />
+          Don't worry, we will hide it on all inbound and outbond <br />
+          calls for your protection
```
Comment (BradenM) [inline src/components/phone/CallerIDEditCard.vue]: See lang change requests in #424

File: src/components/phone/CallerIDEditCard.vue [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      </div>
+      <!-- Footer -->
+      <div class="flex justify-around"></div>
+      <div slot="footer" class="flex p-1 justify-center">
+        <base-button
+          variant="solid"
+          class="px-3 py-2"
+          :action="
+            () => {
+              isShowingModal = false;
```
Comment (BradenM) [inline src/components/phone/CallerIDEditCard.vue]: :P

**Comment by vishk1** [comment]
image

--fixed it

**Pull request merged**